### PR TITLE
Implement changes on calendar page

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: myuw/static/js/card/calendars/academic_calendar.js [hunk 1]
```diff
@@ -82,8 +85,8 @@ var AcademicCalendar = {
     },
```
Comment (klpittman) [inline myuw/static/js/card/calendars/academic_calendar.js]: @fanglinfang My changes in this file are all in the `filter_all` and `filter_breaks` functions.

File: myuw/templates/calendar.html [hunk 2]
```diff
@@ -14,17 +14,21 @@
```
Comment (klpittman) [inline myuw/templates/calendar.html]: @fanglinfang My change on this file is on **line 12**. Changing the attribute `aria-labelledby`.

File: myuw/templates/handlebars/academic_calendar.html [hunk 3]
```diff
@@ -1,60 +0,0 @@
-	{% load templatetag_handlebars %}
-    {% tplhandlebars "calendar_events" %}
-    <div class="container academic_cal-container">
-
-
-        <div class="row">
-            <div class="col-md-8">
-
-                <div class="acal-main">
-                     <h2 id="main" class="heading-page">Academic Calendar</h2>
```
Comment (klpittman) [inline myuw/templates/handlebars/academic_calendar.html]: @fanglinfang in this file, I changed quite a bit of HTML starting here on **line 10 - line 26** and another change on **line 39**.

**Comment by hlee2** [comment]
@fanglinfang Stylings are applied.

**Pull request merged**